website/blog/2021/11/26/apache-apisix-committer-experience.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contributer to Committer journey @Apache APISIX"
+author: "Shivam Singh"
+authorURL: "https://github.com/1502shivam-singh"
+authorImageURL: "https://avatars.githubusercontent.com/u/57267960?v=4"
+keywords:
+- Open Source
+- Apache APISIX
+- Community
+- User story
+description: In this article, Shivam Singh ([@1502shivam-singh](https://github.com/1502shivam-singh)) gives a brief of his journey from starting out in the Apache APISIX community, from a contributor to an Apache member and APISIX committer. His journey can help other people looking to startup in the Apache APISIX community or Open Source in general.
+tags: [Community]
+---
+
+> In this article, Shivam Singh ([@1502shivam-singh](https://github.com/1502shivam-singh)) gives a brief of his journey from starting out in the Apache APISIX community, from a contributor to an Apache member and APISIX committer. His journey can help other people looking to startup in the Apache APISIX community or Open Source in general.
+<!--truncate-->
+
+## Hello reader,
```

Comment:
Maybe changing it to "About me" will be better

Reply:
I feel `Hello reader,` seems more personalised according to the reader and is more humane too. Thoughts on this idea ?

Reply:
I feel 'About Me' or 'Introduction' is better.

Reply:
Okay, seems pretty decent.